multiwallet.go

```diff
@@ -603,17 +611,19 @@ func (mw *MultiWallet) ChangePrivatePassphraseForWallet(walletID int, oldPrivate
 		return translateError(err)
 	}
 
-	decryptedSeed, err := decryptWalletSeed(newPrivatePassphrase, wallet.EncryptedSeed)
-	if err != nil {
-		return err
-	}
+	if wallet.EncryptedSeed != nil {
+		decryptedSeed, err := decryptWalletSeed(newPrivatePassphrase, wallet.EncryptedSeed)
```

Comment:
Shouldn't the encrypted seed be decrypted with the old passphrase first, then re-encrypted with the new passphrase?